File: (7 kyu) Complementary DNA/(7 kyu) Complementary DNA.ts
export class Kata {
  static dnaStrand(dna: string) {
    return dna
      .replace(/A/g, 't')
      .replace(/T/g, 'a')
      .replace(/C/g, 'g')
      .replace(/G/g, 'c')
      .toUpperCase();
  }
}

File: (7 kyu) Complementary DNA/(7 kyu) Complementary DNA.go
package kata

func DNAStrand(dna string) string {
  var dict = map[string]string{
    "A": "T", "T": "A", "C": "G", "G": "C",
  }
  res := ""
  
  for _, v := range dna {
    res = res + dict[string(v)]
  }
  
  return res
}
